extension/runtime/src/main/java/io/quarkiverse/temporal/Constants.java

```diff
@@ -0,0 +1,5 @@
+package io.quarkiverse.temporal;
+
+public class Constants {
```

Comment:
This could be an Interface too if you didn't want to make it a class or move this constant right to TenporalProcessor?

Reply:
As a note an Interface makes it not instantiable

Reply:
Let me make this an interface, there is at least one occurrence in the runtime package so I think it's best if it is defined here